src/metabase/lib/js/metadata.cljs

```diff
@@ -265,17 +265,17 @@
   `:internal`."
   [dimensions]
   (when-let [dimension (m/find-first (fn [dimension]
-                                       (#{"external" "internal"} (object-get dimension "type")))
```

Comment:
`dimension` is a CLJS map here, not JS object; that's why all `object-get` calls return `nil`